src/renderer/src/chat.ts

```diff
@@ -75,7 +99,7 @@ export class ChatView {
     const generator = yieldStream(stream)
 
     const setResponse = (val) => {
-      this.liveMessage.textContent = val
+      this.liveMessageContent.innerHTML = marked.parse(val)
```

Comment:
```suggestion
      this.liveMessageContent.innerHTML = marked.parse(val, {breaks: true})
```

This will tend to be more compatible with how ChatGPT outputs text, since it often doesn't understand the double line break requirement